src/main/resources/templates/fragments/fieldErrorRow.html

```diff
@@ -14,10 +14,16 @@
                           th:text="${e}"
                           class="govuk-error-message">
                     </span>
+                    <span th:if="${!#fields.hasErrors('__${currentField}__')}">&nbsp;</span>
                 </div>
                 <div class="govuk-grid-column-one-quarter">
-                    <!-- This fragment will be expanded to support a previous field -->
-                    &nbsp;
+                    <span th:if="${#fields.hasErrors('__${previousField}__')}"
+                          th:id="${previousField} + '-errorId'"
+                          th:each="e : ${#fields.errors('__${previousField}__')}"
+                          th:text="${e}"
+                          class="govuk-error-message">
+                    </span>
+
```

Comment:
Do we need an empty span here if there are no field errors for the previous field?